### PR TITLE
Add support for 'Activate on new events' option in Error Log

### PR DESCRIPTION
**Pull request by tsedmik**

### CODE REVIEW

File: plugins/org.eclipse.reddeer.eclipse/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -98,3 +98,4 @@ Export-Package: org.eclipse.reddeer.eclipse,
  org.eclipse.reddeer.eclipse.debug.ui.views.variables,
  org.eclipse.reddeer.eclipse.topmenu,
  org.eclipse.reddeer.eclipse.epp.logging.aeri.ide.dialogs
+Import-Package: org.jboss.reddeer.swt.impl.menu
```
Comment (apodhrad) [inline plugins/org.eclipse.reddeer.eclipse/META-INF/MANIFEST.MF]: This looks like an incorrect package

File: plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/ui/views/log/LogView.java [hunk 2]
```diff
@@ -31,6 +28,10 @@
 import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
 import org.eclipse.reddeer.swt.impl.tree.DefaultTree;
 import org.eclipse.reddeer.workbench.impl.view.WorkbenchView;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.widgets.Widget;
+import org.eclipse.ui.internal.views.log.SharedImages;
+import org.jboss.reddeer.swt.impl.menu.ViewMenu;
```
Comment (apodhrad) [inline plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/ui/views/log/LogView.java]: Incorrect package

File: plugins/org.eclipse.reddeer.eclipse/META-INF/MANIFEST.MF (hunk 1 above)
Comment (rawagner) [inline plugins/org.eclipse.reddeer.eclipse/META-INF/MANIFEST.MF]: also please do not use Import-Package but Require-Bundle. 
Require-Bundle already contains org.eclipse.reddeer.swt so there is no need for this line at all.

File: tests/org.eclipse.reddeer.eclipse.test/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -22,4 +22,5 @@ Require-Bundle: org.eclipse.reddeer.go;bundle-version="[2.0,2.1)",
  org.eclipse.epp.logging.aeri.ide
 Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Import-Package: org.hamcrest.core
+Import-Package: org.hamcrest.core,
+ org.jboss.reddeer.swt.impl.menu
```
Comment (rawagner) [inline tests/org.eclipse.reddeer.eclipse.test/META-INF/MANIFEST.MF]: org.eclipse.reddeer.go is already in Require-Bundle so no need for reddeer.swt in Import-Package

**Comment by tsedmik** [comment]
I'm sorry for incorrect packages and needless imports. It should be fixed now ;-).

**Comment by apodhrad** [comment]
testPR

**Comment by apodhrad** [comment]
pushed to master

**Pull request closed without merging**